engine/Assets/Scripts/UI/Dynamic/Modals/Spawning/AddRobotModal.cs

```diff
@@ -40,10 +40,12 @@ public class AddRobotModal : ModalDynamic {
                 }
             });
 
-            var chooseRobotDropdown = MainContent.CreateDropdown()
-                                          .SetOptions(_files.Select(x => Path.GetFileName(x)).ToArray())
-                                          .AddOnValueChangedEvent((d, i, data) => _selectedIndex = i)
-                                          .SetTopStretch<Dropdown>();
+            var chooseRobotDropdown =
+                MainContent.CreateDropdown()
+                    .SetOptions(
+                        _files.Select(x => Path.GetFileName(x).Substring(0, Path.GetFileName(x).Length - 5)).ToArray())
```

Comment:
I feel like we could get some unintended consequences from doing this. However all the files should be `.mira` anyways so I think it should be fine.

Reply:
I'd use this instead: https://learn.microsoft.com/en-us/dotnet/api/system.io.path.getfilenamewithoutextension?view=net-7.0

That being said, I changed where this code is stored in my multiplayer branch it prolly won't matter.